bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -83,6 +95,12 @@ public String getThankYouMessage() {
         return thankyouMessage;
     }
 
+    @Override
+    @Nullable
+    public String getThankYouOption() {
+        return thankYouOption;
+    }
+
```

Comment:
thank you option is not part of schema.

Reply:
how do we handle if author wants to show message then in v2 on form submission?